### ISSUE TITLE
Replication status shows Error

### ISSUE THREAD

**Issue by BlaecHrim** [peterpakos/ipa_check_consistency#14]
On CentOS 7.3 (ipa-server-4.4.0-14) shows all Replication Status as "Error":

```
FreeIPA servers:    lambic    ipa       sahti     STATE
=======================================================
Active Users        32        32        32        OK   
Stage Users         0         0         0         OK   
Preserved Users     3         3         3         OK   
User Groups         15        15        15        OK   
Hosts               22        22        22        OK   
Host Groups         8         8         8         OK   
HBAC Rules          10        10        10        OK   
SUDO Rules          8         8         8         OK   
DNS Zones           3         3         3         OK   
LDAP Conflicts      NO        NO        NO        OK   
Ghost Replicas      NO        NO        NO        OK   
Anonymous BIND      YES       YES       YES       OK   
Replication Status  ipa Error lambic Erroripa Error 
                              sahti Error          
=======================================================
```

The outputs of ipa-replica-manage:

[root@lambic ~]# ipa-replica-manage list -v ipa.linux.otaverkko.fi
lambic.linux.otaverkko.fi: replica
  last init status: None
  last init ended: 1970-01-01 00:00:00+00:00
  last update status: Error (0) Replica acquired successfully: Incremental update succeeded
  last update ended: 2016-12-13 15:08:47+00:00
sahti.linux.otaverkko.fi: replica
  last init status: None
  last init ended: 1970-01-01 00:00:00+00:00
  last update status: Error (0) Replica acquired successfully: Incremental update succeeded
  last update ended: 2016-12-13 15:08:47+00:00

[root@ipa ~]# ipa-replica-manage list -v ipa.linux.otaverkko.fi
lambic.linux.otaverkko.fi: replica
  last init status: None
  last init ended: 1970-01-01 00:00:00+00:00
  last update status: Error (0) Replica acquired successfully: Incremental update succeeded
  last update ended: 2016-12-13 15:41:22+00:00
sahti.linux.otaverkko.fi: replica
  last init status: None
  last init ended: 1970-01-01 00:00:00+00:00
  last update status: Error (0) Replica acquired successfully: Incremental update succeeded
  last update ended: 2016-12-13 15:41:22+00:00

[root@sahti ~]# ipa-replica-manage list -v ipa.linux.otaverkko.fi
lambic.linux.otaverkko.fi: replica
  last init status: None
  last init ended: 1970-01-01 00:00:00+00:00
  last update status: Error (0) Replica acquired successfully: Incremental update succeeded
  last update ended: 2016-12-13 15:41:35+00:00
sahti.linux.otaverkko.fi: replica
  last init status: None
  last init ended: 1970-01-01 00:00:00+00:00
  last update status: Error (0) Replica acquired successfully: Incremental update succeeded
  last update ended: 2016-12-13 15:41:35+00:00

**Comment by peterpakos** [comment]
It looks like the script needs some TLC to fully support CentOS 7.3.

**Comment by peterpakos** [comment]
Fixed by 37c289e.
